scripts/validate_docstrings.py

```diff
@@ -499,7 +499,10 @@ def validate_one(func_name):
             if not rel_desc:
                 errs.append('Missing description for '
                             'See Also "{}" reference'.format(rel_name))
-
+            if rel_name.startswith('pandas.'):
+                errs.append('{} in `See Also` section does not '
+                            'need `pandas` prefix, use {} instead.'
+                            .format(rel_name, rel_name.replace('pandas.', '')))
```

Comment:
I'd use `rel_name[len('pandas.')]` instead of the replace. I don't think in practice will make a difference, but in a case like `pandas.somemod.pandas.MyClass` the replace wouldn't work as expected.

Reply:
Sure. I will update it right away.